uint8 id
uint8 ledEnable

uint8 LED_OFF = 0
uint8 LED_ON = 1
